kolibri/core/notifications/api.py

```diff
@@ -285,6 +300,16 @@ def num_correct(examlog):
     )
 
 
+def num_answered(exam_log):
+    return (
+        exam_log.attemptlogs.values_list("item")
```

Comment:
Hrm, it's an edge case, but there's no guarantee that item ids are unique within quizzes. This is because they are drawn from multiple exercises, where within that exercise each item ids is guaranteed to be unique.

Setting line 305 to:
`exam_log.attemptlogs.values_list("item", "content_id")`
is probably the way to avoid this (admittedly unlikely) edge case.

Reply:
I see that this is also from the `num_correct` function above, so we should probably correct that too.

Reply:
Thanks for that - I've updated both functions accordingly